src/Tracker/onfidoTracker.tsx

```diff
@@ -53,6 +54,40 @@ const source_metadata = {
 
 const stepsArrToString = () => steps?.map((step) => step['type']).join()
 
+const keys = [
+  'documentTypes',
+  'showCountrySelection',
+  'forceCrossDevice',
+  'useLiveDocumentCapture',
+  'uploadFallback',
+  'country',
+  'requestedVariant',
+  'useMultipleSelfieCapture',
+  'photoCaptureFallback',
+  'retries',
+  'useUploader',
+  'useWebcam',
+]
+
+const cleanStepsForConfig = (steps: StepConfig[]): StepConfig[] =>
+  steps.map(
+    (step): StepConfig => {
+      const options = pick(
+        { ...step.options } as Partial<Record<string, unknown>>,
+        keys
+      )
+      const newStep: StepConfig = { ...step }
+
+      if (Object.keys(options).length) {
+        newStep.options = options
```

Comment:
Ideally we should convert the keys to snake case before sending the Looker, so they are more readable

Reply:
Good to know. I converted the first level of `step.options` to snake_case. The second level properties are already in snake_case according to our documentation.